cli/js/headers_test.ts

```diff
@@ -22,14 +23,16 @@ test(function newHeaderTest(): void {
   }
 });
 
-const headerDict = {
+const headerDict: Record<string, string> = {
   name1: "value1",
   name2: "value2",
   name3: "value3",
+  // @ts-ignore
```

Comment:
If undefined values are valid, the types should reflect this instead of ignoring it.

Reply:
These values ultimately end up being used to build a `Headers` object. Again, TypeScript's `lib/lib.dom.d.ts` definition for this type wouldn't accept `undefined` or `null`, so reflecting this would mean diverging from those types.

Reply:
Understood.